src/main/daml/Daml/Finance/Interface/Instrument/Standard/Instrument.daml

```diff
@@ -0,0 +1,39 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Interface.Instrument.Standard.Instrument where
```

Comment:
I'd favor `Token` instead of `Standard` as the name here. `Standard` kind of has this "standardized" co-notation, which isn't really fitting here.

Reply:
I wasn't sure how to name it, I previously had `Basic` which I didn't like much either. I agree `Standard` isn't great as it has multiple meanings which we may not want associated with our instrument. 

I'll change it to `Token`.